railties/lib/rails/generators/rails/app/templates/config/initializers/callback_terminator.rb

```diff
@@ -0,0 +1,4 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Just remove the file on this method https://github.com/rails/rails/blob/7c6861b0bc85077ed1f1dcb4a919de0f1b8d490d/railties/lib/rails/generators/rails/app/app_generator.rb#L89 after the `config` call.